pkg/source/vmware/client.go

```diff
@@ -123,6 +123,23 @@ func (c *Client) Verify() error {
 	return nil
 }
 
+func (c *Client) PreFlightChecks(vm *migration.VirtualMachineImport) (err error) {
+	f := find.NewFinder(c.Client.Client, true)
+	dc := c.dc
+	if !strings.HasPrefix(c.dc, "/") {
+		dc = fmt.Sprintf("/%s", c.dc)
+	}
+	for _, nm := range vm.Spec.Mapping {
+		// The path looks like `/<datacenter>/network/<folder>/<network-name>`.
+		path := filepath.Join(dc, "/network", vm.Spec.Folder, nm.SourceNetwork)
```

Comment:
network objects exist at the dc level and are not scoped to the vm folder. I do not think we need `vm.Spec.Folder` here.

sample from one environment
```
(⎈|default:N/A)➜  ~ govc find / -type n
/Datacenter/network/VM Network
```

Reply:
rest of the changes look good

Reply:
When i look into the code of [Finder.Network](https://github.com/vmware/govmomi/blob/5aff45ed941ee6c5648b1264b495a8108b36c999/find/finder.go#L798) there is a cluster path example and the text is mentioning `network folder`.